clang/lib/DPCT/ASTTraversal.cpp

```diff
@@ -3398,6 +3400,9 @@ void EnumConstantRule::runRule(const MatchFinder::MatchResult &Result) {
       EnumName == "cudaComputeModeExclusiveProcess") {
     handleComputeMode(EnumName, E);
     return;
+  } else if (EnumName == "cudaStreamCaptureStatusInvalidated") {
+    report(E->getBeginLoc(), Diagnostics::KNOWN_UNSUPPORTED_TYPE, true,
```

Comment:
It is not a type, you need report error like this:
```c++
// CHECK:     /*
// CHECK:     DPCT1007:{{[0-9]+}}: Migration of CUDNN_BATCHNORM_SPATIAL_PERSISTENT is not supported.
// CHECK:     */
// CHECK:     dpct::dnnl::batch_normalization_mode m = dpct::dnnl::batch_normalization_mode::spatial;
    cudnnBatchNormMode_t m = CUDNN_BATCHNORM_SPATIAL_PERSISTENT;
```

Reply:
Done.